src/utils/rates.js

```diff
@@ -52,3 +52,17 @@ export const getAssetValueInFiat = (
 
   return wrapBigNumber(assetValue)?.times(rate).toNumber();
 };
+
+export const getAssetValueFromFiat = (
+  fiatValue: ?BigNumber | string | number,
```

Comment:
Does it has something to do with BigNumber not being present or we can just replace this with `Value`?

Reply:
It's the same as `?Value` or `BigNumber | string | number | null | void`

Reply:
Changed